docs/book/src/introduction.md

```diff
@@ -0,0 +1,50 @@
+# The Cluster API Book
+
+**Note:** Impatient readers may head straight to [Quick
+Start](quick-start.md).
+
+**Using Cluster API v1alpha1? Check the [legacy
+documentation](https://release-0-1.cluster-api.sigs.k8s.io)**
+
+# What is the Cluster API?
+
+![](images/introduction.png)
```

Comment:
I think we should use the Turtles here instead, since that is more of a logo for the project vs this graphic which is more of a graphic for a particular use case.

Reply:
Oh interesting, I thought they were both logos. I used the turtles for the favicon though. I can switch if you prefer given that this is the introduction

Reply:
Fixed!